src/main/java/ca/corefacility/bioinformatics/irida/ria/web/linelist/LineListController.java

```diff
@@ -349,4 +359,18 @@ This field is to display to the user any notification icons that they might have
 
 		return fields;
 	}
+
+	/**
+	 * Check to see if the currently logged in user has permission to edit {@link MetadataEntry} on the current project
+	 *
+	 * @param project {@link Project}
+	 * @return {@link Boolean} true if user can edit on the current project
+	 */
+	private boolean canUserEdit(Project project) {
+		Authentication authentication = SecurityContextHolder.getContext()
+				.getAuthentication();
+		User user = (User) authentication.getPrincipal();
+		return user.getSystemRole()
+				.equals(Role.ROLE_ADMIN) || projectOwnerPermission.isAllowed(authentication, project);
```

Comment:
Should be no need to add the `ROLE_ADMIN` check here.  The permission will return true if the user is an admin (and the project is local).

Reply:
Removed in b5e9275e013b023cb27469613c83b11cbc747539